src/django_perf_rec/cache.py

```diff
@@ -10,7 +11,8 @@
 
 
 class CacheOp(Operation):
-    def __init__(self, alias, operation, key_or_keys):
+    def __init__(self, alias, operation, key_or_keys, trace_back):
```

Comment:
I've fixed the name `trace_back` to `traceback` throughout the code - I've always seen it as one word, and wiktionary agrees: https://en.wiktionary.org/wiki/traceback